charts/rollout-app/Chart.yaml

```diff
@@ -2,14 +2,14 @@ apiVersion: v2
 name: rollout-app
 description: Argo Rollout-based Application Helm Chart
 type: application
-version: 0.7.4
+version: 0.7.5
```

Comment:
Given the major change in the `istio-alerts` behavior, we should bump the minor versions here I believe.

Reply:
updated